tensorflow/contrib/eager/python/saver.py

```diff
@@ -93,24 +93,25 @@ def restore_variables_on_create(save_path, map_func=None):
     old_init = getattr(resource_variable_ops.ResourceVariable,
                        "_init_from_args", None)
     assert old_init, "ResourceVariable misses _init_from_args method."
-    setattr(resource_variable_ops.ResourceVariable, "_init_from_args",
-            _init_from_checkpoint)
-    setattr(resource_variable_ops.ResourceVariable, "_old_init", old_init)
-    setattr(resource_variable_ops.ResourceVariable, "_map_func",
-            map_func_wrapper)
-    setattr(resource_variable_ops.ResourceVariable, "_ckpt_var_cache",
-            ckpt_var_cache)
+    # pylint: disable=protected-access
+    resource_variable_ops.ResourceVariable._init_from_args = \
```

Comment:
remove the \

Reply:
use parens

Reply:
Sorry, I missed your comment above. Fixed in c82356c2096f911a28b94c14ff6bb055bf5361c6